lib/pbench/agent/base.py

```diff
@@ -21,10 +21,14 @@ def __init__(self, context):
         self.config = PbenchAgentConfig(self.context.config)
         self.name = os.path.basename(sys.argv[0])
 
-        self.pbench_run = self.config.pbench_run
+        env_pbench_run = os.environ.get("pbench_run")
+        self.pbench_run = pathlib.Path(
+            self.config.pbench_run if env_pbench_run is None else env_pbench_run
+        )
```

Comment:
Rather than testing to see if `env_pbench_run` is `None` (which would mean that it was not defined), I suggest testing it for "truthiness", e.g.:
```
        self.pbench_run = pathlib.Path(
            env_pbench_run if env_pbench_run else self.config.pbench_run
        )
```
In this way, if it exists but has a null or empty value, we will also ignore it.